src/commands/internal/create-release-announcement.ts

```
@@ -79,13 +86,36 @@ export async function getReleaseAnnouncement(mode: string): Promise<string> {
     })
     .join('\n');
 
+  const packageVersionChangeArray = [];
+  if (additionalVersionAnnouncements != null) {
+    const startCommit = getStartCommit(startDate, getGitBranch());
+    const diff = getDiffFromStartToCurrentHead(startCommit, 'package.json').split('\n');
+
+    additionalVersionAnnouncements.forEach((packageName: string) => {
+      const filteredByPackage = diff.filter((diffEntry) => diffEntry.includes(`"${packageName}":`));
+      const entryStartsWithMinus = filteredByPackage.some((entry) => entry.trim().startsWith('-'));
+      const entryStartsWithPlus = filteredByPackage.some((entry) => entry.trim().startsWith('+'));
+
+      if (filteredByPackage.length === 2 && entryStartsWithMinus && entryStartsWithPlus) {
+        const packageVersionObject = getPackageVersionObject(filteredByPackage, packageName);
+        packageVersionChangeArray.push(packageVersionObject);
+      }
```

Das hier versagt, wenn ich einen Package-Namen angebe, der nicht vorkommt, und "warnt" mich auch nicht, wenn die Dependency entfernt wird, sondern endet in einem "silent fail" (es findet sich nirgendwo ein Hinweis, dass das, was ich angegeben habe, keinen Sinn ergibt).

Bitte folgende Fälle mit berücksichtigen:

- Wenn ich eine Dependency angebe, die es nicht gibt, muss ich das im Release Announcement sehen.
- Wenn ich eine Dependency entferne, die ich angegeben habe, muss ich das im Release Announcement sehen.